ur_controllers/controller_plugins.xml

```diff
@@ -4,4 +4,19 @@
       The force torque state controller publishes the current force and torques
     </description>
   </class>
+  <class name="ur_controllers/SpeedScalingStateController" type="ur_controllers::SpeedScalingStateController" base_class_type="controller_interface::ControllerInterface">
+  <description>
+    This controller publishes the readings of all available speed scaling factors.
+  </description>
+  </class>
+  <class name="position_controllers/ScaledJointTrajectoryController" type="position_controllers::ScaledJointTrajectoryController" base_class_type="controller_interface::ControllerBase">
+  <description>
+    Scaled velocity-based joint trajectory controller
+  </description>
+  </class>
+  <class name="velocity_controllers/ScaledJointTrajectoryController" type="velocity_controllers::ScaledJointTrajectoryController" base_class_type="controller_interface::ControllerBase">
```

Comment:
This PR is oriented towards SpeedScalingStateController. Remove the others as they will be added in additional PR.

Reply:
see above